### PR TITLE
Migrate dispatchr to monorepo

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: packages/dispatchr/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Contributing Code to `dispatchr`
+-------------------------------
+
+Please be sure to sign our [CLA][] before you submit pull requests or otherwise contribute to `dispatchr`. This protects developers, who rely on [BSD license][].
+
+[BSD license]: https://github.com/yahoo/dispatchr/blob/master/LICENSE.md
```
Comment (redonkulus) [inline packages/dispatchr/CONTRIBUTING.md]: Update license link.

File: packages/dispatchr/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Dispatchr
```
Comment (redonkulus) [inline packages/dispatchr/README.md]: Need to update all links and badges

File: packages/dispatchr/docs/dispatchr.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Dispatchr API
```
Comment (redonkulus) [inline packages/dispatchr/docs/dispatchr.md]: Double check all links

**Pull request merged**